lib/improver/feels_like_temperature.py

```diff
@@ -0,0 +1,240 @@
+    # convert temperature units
+    temperature.convert_units('celsius')
+    # convert wind speed to km/h
+    wind_speed.convert_units('km h-1')
+    eqn_component = (wind_speed.data)**0.16
+    wind_chill_data = (13.12 + 0.6215*temperature.data-11.37*eqn_component
+                       + 0.3965*temperature.data*eqn_component)
+    wind_chill = temperature.copy(data=wind_chill_data)
+    wind_chill.rename("wind_chill")
+    wind_chill.convert_units('K')
+    temperature.convert_units(temp_units)
+    wind_speed.convert_units(wind_units)
+    return wind_chill
+
+
+def calculate_apparent_temperature(temperature, wind_speed,
+                                   relative_humidity, pressure):
+    """
+    Calculates the apparent temperature from 10 m wind speed, temperature
+    and actual vapour pressure using the linear regression equation
+    for shade described in A Universal Scale of Apparent Temperature,
+    Steadman, 1984.
+
+    The paper calculates apparent temperature for wind speeds up to 20 m/s.
+    Here, the apparent temperature regression equation has been used for all
+    wind speeds.
+
+    The function looks up a value for the saturation vapour pressure of
+    water vapour using the temperature and a table of values. These tabulated
+    values are found using lookup_svp and are corrected to the saturated
+    vapour pressure in air using pressure_correct_svp both from the
+    WetBulbTemperature plugin which makes use of the Goff-Gratch method.
+
+    Args:
+      temperature (iris.cube.Cube):
+        Cube of air temperatures (K)
+
+      wind_speed (iris.cube.Cube):
+        Cube of 10m wind speeds (m/s)
+
+      relative_humidity (iris.cube.Cube):
+        Cube of relative humidities (fractional)
+
+      pressure (iris.cube.Cube):
+        Cube of air pressure (Pa)
+
+    Returns:
+      apparent_temperature (iris.cube.Cube):
+        Cube of apparent temperatures (K)
+
+    References:
+      Steadman, R. (1984). A Universal Scale of Apparent Temperature.
+      Journal of Climate and Applied Meteorology, 23(12), pp.1674-1687
+    """
+    # take a copy of each cube's orignial units
+    temp_units = temperature.copy().units
+    wind_units = wind_speed.copy().units
+    pressure_units = pressure.copy().units
+    relative_humidity_units = relative_humidity.copy().units
+
+    # ensure units are correct
+    wind_speed.convert_units('m s-1')
+    pressure.convert_units('Pa')
+    relative_humidity.convert_units('1')
+    temperature.convert_units('K')
+    # look up saturated vapour pressure
+    svp = WetBulbTemperature().lookup_svp(temperature)
+    # convert to SVP in air
+    svp = WetBulbTemperature().pressure_correct_svp(
+        svp, temperature, pressure)
+    # convert temperature units
+    temperature.convert_units('celsius')
+    # calculate actual vapour pressure
+    # and convert relative humidities to fractional values
+    avp_data = svp.data*relative_humidity.data
+    avp = svp.copy(avp_data)
```

Comment:
To fit in with the rest of the code, shouldn't this be? 

`avp = svp.copy(data=avp_data)`

Reply:
I agree.

Reply:
Changed.